src/api/routes.ts

```diff
@@ -638,4 +642,5 @@ export const API_ROUTES = {
   buildPutItemGeolocationRoute,
   buildGetAddressFromCoordinatesRoute,
   buildDeleteItemThumbnailRoute,
+  buildGetEmbeddedLinksMetadata: buildGetEmbeddedLinkMetadata,
```

Comment:
Is it possible to export it with the same name ?
```suggestion
  buildGetEmbeddedLinkMetadata,
```